app/src/main/java/com/mstudio/superheroarch/presentation/DetailsViewModel.kt

```
@@ -1,40 +1,84 @@
             }
         }
     }
 
+    private fun fetchTMDBEpisodeDetails(seasonNumber: Int, episodeNumber: Int) {
+        viewModelScope.launch {
+            try {
+                val episodeTMDBResult = getEpisodeTMDBDetailsUseCase(seasonNumber, episodeNumber)
+                episodeTMDBResult?.let {
+                    view.displayEpisodeRatingAndImage(it.rating, "") // Asegúrate de agregar la imagen si es necesario
+                }
+            } catch (e: Exception) {
```

detekt

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)